hack/debug.sh

```diff
@@ -45,20 +71,15 @@ do_master () {
 
     logmaster=$logdir/master
     mkdir -p $logmaster
-    echo_and_eval journalctl --boot >& $logmaster/journal-full
-    echo_and_eval journalctl -u $aos_master_service >& $logmaster/journal-openshift
-    echo_and_eval systemctl show $aos_master_service >& $logmaster/systemctl-show
-    echo_and_eval nmcli --nocheck -f all dev show >& $logmaster/nmcli-dev
-    echo_and_eval nmcli --nocheck -f all con show >& $logmaster/nmcli-con
-    echo_and_eval head -1000 /etc/sysconfig/network-scripts/ifcfg-* >& $logmaster/ifcfg
-    echo_and_eval ip addr show >& $logmaster/addresses
-    echo_and_eval ip route show >& $logmaster/routes
-    echo_and_eval iptables-save >& $logmaster/iptables
-    echo_and_eval cat /etc/hosts >& $logmaster/hosts
-    echo_and_eval cat /etc/resolv.conf >& $logmaster/resolv.conf
-    echo_and_eval oc get nodes -o json >& $logmaster/nodes
-    echo_and_eval oc get pods --all-namespaces -o json | filter_env >& $logmaster/pods
-    echo_and_eval oc get services --all-namespaces -o json >& $logmaster/services
+
+    # Log the generic system stuff
+    log_system $logmaster
+
+    # And the master specific stuff
+    echo_and_eval  oc get nodes                     -o json           &> $logmaster/nodes
+    echo_and_eval  oc get pods     --all-namespaces -o json           &> filter_env >& $logmaster/pods
+    echo_and_eval  oc get services --all-namespaces -o json           &> $logmaster/services
+    echo_and_eval  oc get routes   --all-namespaces -o json           &> $logmaster/aos_routes
```

Comment:
Can we also capture output of these commands:
oc get clusternetwork -o json
oc get hostsubnets -o json
oc get netnamespaces -o json